src/rebar_prv_shell.erl

```diff
@@ -331,8 +331,8 @@ reread_config(State) ->
     case find_config(State) of
         no_config ->
             ok;
-        ConfigList ->
-            _ = rebar_utils:reread_config(ConfigList),
+        Config ->
+            _ = rebar_utils:reread_config([Config]),
```

Comment:
Because previously, consult_config returned a list of configs separated by included files, and now it returns, necessarily, one merged config to preserve order (as preserving order is the correct behaviour). This is a compatibility change, although maybe a misplaced one.